zarr/storage.py

```diff
@@ -1990,21 +1990,26 @@ def __setstate__(self, state):
         self.client = BlockBlobService(self.account_name, self.account_key,
                                        **self.blob_service_kwargs)
 
-    @staticmethod
-    def _append_path_to_prefix(path, prefix):
-        return '/'.join([normalize_storage_path(prefix),
-                         normalize_storage_path(path)])
+    def _append_path_to_prefix(self, path):
```

Comment:
Just wondering, what happens here if `self.prefix` is `None` (default)?

Reply:
During instantiation in the `__init__` function  any `prefix` supplied is passed to the `zarr.util.normalize_storage_path` function before storing it in `self.prefix`, so if prefix is `None` it will be stored as `''` in `self.prefix`

Reply:
Ah cool, thanks. In which case maybe the default value for the prefix parameter could the empty string rather than None, might be a little more obvious. Also if prefix is normalised in the constructor then probably could remove calls to normalize self.prefix in the methods.